SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -3151,6 +3151,12 @@
                 "zlib",
                 "zsh"
             ]
+        },
+        "Rancher": {
```

Comment:
Just to be clear, you took the SPEC file from something called "Rancher" and are using it under the Apache license?

Reply:
Correct, the SPEC file is modified from the one in the official k3s github repo. Rancher is the company which created and maintains k3s (and by extension the original SPEC file).